src/app.js

```diff
@@ -840,7 +843,7 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   })
 
   function startdebugging (txHash) {
-    self.event.trigger('debuggingRequested', [])
+    self._components.righthandpanel.showDebugger()
```

Comment:
not sure that is needed to replace that now

Reply:
is that possible to roll back that?

Reply:
i would prefer not to.
The previous line of code triggered inside of `app.js` an event that bubbles down to children components which i perceive as going against our best practices.

To me: `data is passed down` and `events bubble up`.
The only time an "event bubbles down" is when an event is triggered by a child or leaf component and the event listener is/was passed down by app.js to other child or leaf components to listen to it.

The idea behind that:
* A leaf/child component can listen to an event caused by another leaf/child component somwhere else

...but this is not the case here.

**So considering that, would you agree to leave this change as is maybe?**